Ramos_HwAssignments/Ramos_ClassAssignment_BarcodeScanner/BarcodeScannerApp.tsx

```diff
@@ -1,17 +1,62 @@
 import { Text, View } from 'react-native';
+import { FontAwesomeIcon } from '@fortawesome/react-native-fontawesome';
+import { faCircle } from '@fortawesome/free-regular-svg-icons'
 import { NavigationContainer, useNavigation, useRoute, RouteProp, StackActions } from '@react-navigation/native';
-import { createStackNavigator } from '@react-navigation/stack'
+import { createStackNavigator } from '@react-navigation/stack';
+import { createBottomTabNavigator } from '@react-navigation/bottom-tabs';
 import { BarcodeScanner } from './BarcodeScanner';
+import { ProductDetail } from './ProductDetail';
+import { Favorites } from './Favorites';
 
-export type StackParamList = {
+export type ScannerStackParamList = {
   BarcodeScanner: undefined;
+  ProductDetail: { data: any };
 };
-const Stack = createStackNavigator<StackParamList>();
 
-export default function WeatherApp() {
+const Tab = createBottomTabNavigator();
+const ScannerStack = createStackNavigator<ScannerStackParamList>();
+const FavoritesStack = createStackNavigator<ScannerStackParamList>();
```

Comment:
`FavoritesStack` should have it's own type, it's not the same as the scanner stack. Theres an error on line 32 for the "name" of the screen